data-analytics/dataflow-python-examples/dataflow_python_examples/data_generation_for_benchmarking.py

```diff
@@ -0,0 +1,485 @@
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument('--schema_file', dest='schema_file', required=False,
+                        help='Schema json file to read. This can be a local file or a file in a \
+                         Google Storage Bucket.')
+
+    parser.add_argument('--input_bq_table', dest='input_bq_table', required=False,
+                        help='Name of BigQuery table to populate.')
+
+    parser.add_argument('--output_bq_table', dest='output_bq_table', required=False,
+                        help='Name of the table to write to BigQuery table.')
+
+    parser.add_argument('--num_records', dest='num_records', required=False,
+                        help='Number of random output records to write to BigQuery table.',
+                        default=10)
+
+    parser.add_argument('--p_null', dest='p_null', required=False,
+                        help='Probability a nullable column is null.',
+                        default=0.2)
+
+    parser.add_argument('--n_keys', dest='n_keys', required=False,
+                        help='Cardinality of key columns.',
+                        default=1000)
+
+    parser.add_argument('--min_date', dest='min_date', required=False,
+                        help='Set earliest possible date for the history represented by this table,\
+                         %Y-%m-%d format.',
+                        default=datetime.date(2018, 1, 1).strftime('%Y-%m-%d'))
+
+    parser.add_argument('--max_date', dest='max_date', required=False,
+                        help='Set latest possible date for the history represented by this table \
+                        %Y-%m-%d format.',
+                        default=datetime.date.today().strftime('%Y-%m-%d'))
+
+    parser.add_argument('--strictly_positive', dest='only_pos', required=False,
+                        help='Dictates if numbers (integers or floats) generated be \
+                        strictly positive.',
+                        default=True)
+
+    parser.add_argument('--max_int', dest='max_int', required=False,
+                        help='Maximum integer.',
+                        default=10 ** 11)
+
+    parser.add_argument('--max_float', dest='max_float', required=False,
+                        help='Maximum float.',
+                        default=float(10 ** 11))
+
+    parser.add_argument('--float_precision', dest='float_precision', required=False,
+                        help='How many digits to the right of the decimal for floats.',
+                        default=2)
+
+    parser.add_argument('--write_disposition', dest='write_disp', required=False,
+                        help='WriteDisposition for BigQuery. Options are WRITE_APPEND, WRITE_EMPTY\
+                             WRITE_TRUNCATE.',
+                        default='WRITE_APPEND')
+
+    data_args, pipeline_args = parser.parse_known_args(argv)
+
+    pipeline_options = PipelineOptions(pipeline_args)
+
+    if data_args.schema_file is None:
```

Comment:
use python booleans:

`if not data_args.schema_file:`

Reply:
Changing to use python boolenans.